apps/aecore/lib/aecore/chain/block.ex

```diff
@@ -2,25 +2,28 @@ defmodule Aecore.Chain.Block do
   @moduledoc """
   Structure of the block
   """
+
   alias Aecore.Chain.Block
   alias Aecore.Chain.Header
   alias Aecore.Tx.SignedTx
   alias Aecore.Chain.BlockValidation
   alias Aeutil.Serialization
 
+  # FIXME: Duplicateof header version
```

Comment:
is this still open to be fixed? If not could you clarify in your comment

Reply:
I will remove this comment probably. I added it here, because block and header version numbers seem are duplicated. We will probably fix that when we fix block serialization. I don't this we are following epoch currently with that.